modules/@apostrophecms/i18n/index.js

```diff
@@ -523,7 +523,15 @@ module.exports = {
               self.namespaces[ns].browser = self.namespaces[ns].browser ||
                 (metadata[ns] && metadata[ns].browser);
               const namespaceDir = path.join(localizationsDir, ns);
+              if (!fs.statSync(namespaceDir).isDirectory()) {
+                // Handle files in the namespace that aren't JSON files
+                continue;
+              }
               for (const localizationFile of fs.readdirSync(namespaceDir)) {
+                if (!localizationFile.endsWith('.json')) {
+                  // A JSON file for the default namespace, already handled
```

Comment:
How would a non-json file in a namespace folder be a JSON file for the default namespace? This check makes sense but the comment has me wondering if things were transposed somehow.

Reply:
Yeah - I think that copilot "helpfully" wrote that comment and I wasn't careful enough in reading it. That conditional makes sure that within the namespace folder only JSON files are being parsed. I'll change the comment.